tvtk/wrapper_gen.py

```diff
@@ -8,24 +8,185 @@
 
 from __future__ import print_function
 
+import re
 import sys
 import vtk
 import types
 import textwrap
 import keyword
 import copy
+from collections import Sequence
+from itertools import chain
 
 # Local imports (these are relative imports because the package is not
 # installed when these modules are imported).
-from .common import (get_tvtk_name, camel2enthought, is_version_58,
-    is_version_62)
+from .common import get_tvtk_name, camel2enthought, is_version_58
+
 from . import vtk_parser
 from . import indenter
 from . import special_gen
 
 PY_VER = sys.version_info[0]
 
 
+def get_trait_def(value, kwargs="enter_set=True, auto_set=False"):
+    """ Return the appropriate trait type, reformatted string and
+    the associated traits meta data for a given `value`
+
+    If a sequence is given, traits.Array is returned instead of
+    traits.Tuple or traits.List
+
+    Parameters
+    ----------
+    value
```

Comment:
You pretty much have to put something, even trivial, I guess? if we run the docgen on it I think it will complain (do we even run the docgen on it?). Good for me.